rae_description/urdf/rae.urdf.xacro

```diff
@@ -353,6 +354,113 @@
     <child link="rae_imu_frame"/>
   </joint>
 
-  <xacro:rae_ros2_control sim_mode="$(arg sim_mode)"/>
+  <xacro:rae_ros2_control sim_mode="$(arg sim_mode)" namespace="$(arg namespace)" />
+
+  <xacro:if value="$(arg sim_mode)">
+    <link name="rae_rgb_camera_sim_frame">
+      <xacro:body mass="0.00001" d="0.0000010" h="0.000003" w="0.00003">
+        <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+      </xacro:body>
+
+    </link>
+
+    <joint name="rae_rgb_camera_sim_frame_joint" type="fixed">
+      <origin xyz="0.0 0 0" rpy="${pi/2} ${-pi/2} 0.0"/>
+      <parent link="rae_rgb_camera_optical_frame"/>
+      <child link="rae_rgb_camera_sim_frame"/>
+    </joint>
+
+    <gazebo reference="rae_rgb_camera_sim_frame">
+      <sensor type="camera" name="rae_rgb_camera">
+        <always_on>true</always_on>
+        <update_rate>30.0</update_rate>  
+        <topic>/rae/right/image_raw</topic>
+        <ignition_frame_id>rae_rgb_camera_optical_frame</ignition_frame_id>
+        <camera>
+          <horizontal_fov>${105.0/180.0*pi}</horizontal_fov>
```

Comment:
Sensor information can be found [here](https://docs-beta.luxonis.com/hardware/products/Rae)

Reply:
Hopefully fixed